app/controllers/staff/reports_controller.rb

```diff
@@ -12,15 +12,17 @@ def index
   end
 
   def show
-    @report = Report.find(id)
-    authorize @report
+    report = Report.find(id)
+    authorize report
+
+    @report_presenter = ReportPresenter.new(report)
```

Comment:
The HTML view references the `@report` instance variable rather than `@report_presenter` so this probably should be updated.

Since `@report` has been renamed, it is no longer be available in the view, so I'm a bit worried no tests broke..!